bibernate-core/src/main/java/com/bobocode/petros/bibernate/session/DefaultSession.java

```diff
@@ -17,51 +19,51 @@ public DefaultSession(final DataSource dataSource, final JdbcQueryManager jdbcQu
 
     @Override
     public <T> T persist(T entity) {
-        return null;
+        return jdbcQueryManager.persist(entity);
     }
 
     @Override
     public <T, ID> T findById(Class<T> type, ID id) {
-        return null;
+        return jdbcQueryManager.find(type, List.of(Restrictions.idEq("id", id)));
```

Comment:
Property name id shouldn't be hardcoded.
EntityUtils.getIdField(type).getName() instead of "id"